omnigibson/systems/micro_particle_system.py

```diff
@@ -393,6 +408,13 @@ class MicroParticleSystem(BaseSystem):
     # Particle system prim in the scene, should be generated at runtime
     system_prim = None
 
+    # Material -- MaterialPrim associated with this particle system
+    _material = None
```

Comment:
hmm are these functionalities actually shared by all subclasses of micro particle systems?
For instance, I thought only Fluid needs to create material. I felt like previously we intentionally placed these functions in different classes (`MicroParticleSysten`, `PhysicalParticalSystem`, `FluidSystem`) for a reason.